test/interactive_setup_api_integration/fixtures/README.md

```diff
@@ -6,27 +6,72 @@ The Elasticsearch HTTP layer keystore is supposed to mimic the PKCS12 keystore t
+  --dns=localhost --dns=localhost.localdomain \
+  --dns=localhost4 --dns=localhost4.localdomain4 \
   --dns=localhost6 --dns=localhost6.localdomain6 \
-  --ip=127.0.0.1 --ip=0:0:0:0:0:0:0:1
-```
+  --ip=127.0.0.1 --ip=0:0:0:0:0:0:0:1 \
+  -days 18250 --out $EE1.p12 --pass storepass
 
-Change the alias of the TrustedCertificateEntry so that it won't clash with the CA PrivateKeyEntry
-```bash
-keytool -changealias -alias ca -destalias cacert -keystore \
-  $KIBANA_HOME/test/interactive_setup_api_integration/fixtures/elasticsearch.p12 \
-  -deststorepass "storepass"
-```
+# Convert P12 keystores to PEM files
+openssl pkcs12 -in $CA_PATH -out $CA1.crt -nokeys -passin pass:castorepass -passout pass:
+openssl pkcs12 -in $CA_PATH -nocerts -passin pass:castorepass -passout pass:keypass | openssl rsa -passin pass:keypass -out $CA1.key
+
+openssl pkcs12 -in $CA2.p12 -out $CA2.crt -nokeys -passin pass:castorepass -passout pass:
+openssl pkcs12 -in $CA2.p12 -nocerts -passin pass:castorepass -passout pass:keypass | openssl rsa -passin pass:keypass -out $CA2.key
+
+openssl pkcs12 -in $EE1.p12 -out $EE1.crt -clcerts -passin pass:storepass -passout pass:
+openssl pkcs12 -in $EE1.p12 -nocerts -passin pass:storepass -passout pass:keypass | openssl rsa -passin pass:keypass -out $EE1.key
+
+# Re-sign intermediate CA cert
+mkdir -p ./tmp
+openssl x509 -x509toreq -in $CA2.crt -signkey $CA2.key -out ./tmp/$CA2.csr
+dd if=/dev/urandom of=./tmp/rand bs=256 count=1
+touch ./tmp/index.txt
+echo "01" > ./tmp/serial
+cp /System/Library/OpenSSL/openssl.cnf ./tmp/
+echo "
+[ tmpcnf ]
+dir             = ./
+certs           = ./
+new_certs_dir   = ./tmp
+crl_dir         = ./tmp/crl
+database        = ./tmp/index.txt
+unique_subject  = no
+certificate     = ./$CA1.crt
+serial          = ./tmp/serial
+crlnumber       = ./tmp/crlnumber
+crl             = ./tmp/crl.pem
+private_key     = ./$CA1.key
+RANDFILE        = ./tmp/rand
+x509_extensions = v3_ca
+name_opt        = ca_default
+cert_opt        = ca_default
+default_days    = 18250
+default_crl_days= 30
+default_md      = sha256
+preserve        = no
+policy          = policy_anything
+" >> ./tmp/openssl.cnf
+
+openssl ca -batch -config ./tmp/openssl.cnf -name tmpcnf -in ./tmp/$CA2.csr -out $CA2.crt
+
+# Convert PEM files back to P12 keystores
+cat $CA2.key $CA2.crt $CA1.crt | openssl pkcs12 -export -name $CA2 -passout pass:castorepass -out $CA2.p12
+cat $EE1.key $EE1.crt $CA1.crt $CA2.crt | openssl pkcs12 -export -name $EE1 -passout pass:storepass -out $EE1.p12
+
+# Verify contents of keystores
+openssl pkcs12 -info -in $CA2.p12 -passin pass:"castorepass" -nodes
+openssl pkcs12 -info -in $EE1.p12 -passin pass:"storepass" -nodes
 
-Import the CA PrivateKeyEntry
-```bash
-keytool -importkeystore \
-  -srckeystore $KIBANA_HOME/packages/kbn-dev-utils/certs/ca.p12 \
-  -srcstorepass "castorepass" \
-  -destkeystore $KIBANA_HOME/test/interactive_setup_api_integration/fixtures/elasticsearch.p12 \
-  -deststorepass "storepass"
```

Comment:
__question:__ Hmm, are you sure we no longer need these instructions to generate proper certificates to test the enrollment flow (not manual configuration flow)? 

I'd expect us to have multiple separate sets of certificates for different test scenarios (and hence instructions): at least one for the manual configuration tests (with intermediate CA) and one for the enrollment flow. For the enrollment flow it'd probably make sense to use the same approach as [Elasticsearch team uses](https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/security/src/test/resources/org/elasticsearch/xpack/security/action/enrollment/README.asciidoc). Or what's your plan?

Reply:
Yes, had to add them back in. Elasticsearch started ok but the enrollment flow doesn't work without CA private key